server/novaboot-shell

```diff
@@ -124,6 +126,22 @@ run_subcommand() {
 	    mkdir -p "$HOME/tftproot"
 	    cd "$HOME/tftproot"
 	    exec "$@";;
+	"rsync-nfsroot "*" --server "*" . .")
+	    shift 1
+	    mkdir -p "$HOME/nfsroot"
+	    cd "$HOME/nfsroot"
+	    exec rsync "$@";;
+	"unfsd" *)
+	    rootdir="$2"
+	    shift 2
+
+	    [ -n "$board_ip" ] || die 'Missing configuration option $board_ip'
```

Comment:
Is it required to be an IP address or hostnames could be supported as well? If possible, I'd prefer that both IP addresses and hostnames can be specified here. Most programs can handle both.

And to be consistent with the rest of the file, I'd prefer checking variables with `${board_ip:?}`, when the variable is first used (the line below). The error message will be generated automatically by the shell.

Reply:
The unfs3 manual says hostnames are supported as well so it should work.